chapters/commonvalidity/draw_dispatch_common.txt

```diff
@@ -215,15 +215,17 @@ ifdef::VK_VERSION_1_1,VK_KHR_sampler_ycbcr_conversion[]
     object must: not use the code:ConstOffset and code:Offset operands
 endif::VK_VERSION_1_1,VK_KHR_sampler_ycbcr_conversion[]
   * [[VUID-{refpage}-None-04115]]
-    If a slink:VkImageView is accessed using code:OpImageWrite as a result
-    of this command, then the code:Type of the code:Texel operand of that
-    instruction must: have at least as many components as the image view's
-    format
+    If as a result of this command a slink:VkImageView is accessed using
+    code:OpImageWrite with an code:OpTypeImage that has an "`Image Format`"
+    operand of code:Unknown, then the code:Type of the code:Texel operand
+    of that instruction must: have at least as many components as the
+    image view's format
```

Comment:
I wonder if it's a little bit misleading including "that has an Image Format operand of Unknown" here. It's not that these are allowed if the operand is not unknown, but rather that you can't have got this far in that case. It might be a little nicer to remove that phrase?

Reply:
I agree, I guess it was easier to have here to justify **why** the other VU is needed, but agree it makes a hard VU to read even harder to read